README.md

```diff
@@ -1,2 +1,103 @@
-# iexec-voucher-subgraph
-Subgraph for iExec Voucher contracts
+# voucher-subgraph
+
+A subgraph indexing `iexec-voucher-contracts`
```

Comment:
We can put a link to the contracts repo.

Reply:
Done